Makefile

```diff
@@ -94,7 +94,17 @@ else
 	GO111MODULE=on go build -o $(GOPATH)/bin/golangci-lint ./vendor/github.com/golangci/golangci-lint/cmd/golangci-lint
 endif
 
-install-tools: install-golangci-lint install-setup-envtest install-go-junit-report
+SHELLCHECK := $(shell command -v shellcheck 2> /dev/null)
+install-shellcheck:
+ifdef SHELLCHECK:
+	@echo "Found shellcheck"
+	shellcheck --version
+else
+	@echo "Installing shellcheck"
+	mkdir -p $(GOPATH)/bin && curl -L https://github.com/koalaman/shellcheck/releases/download/v0.9.0/shellcheck-v0.9.0.linux.x86_64.tar.xz | tar -Jxv --strip-components=1 -C $(GOPATH)/bin shellcheck-v0.9.0/shellcheck
```

Comment:
This is the 3rd flow for this I've seen recently in repos I watch.

- See e.g. https://github.com/openshift/os/pull/1001
- And https://github.com/openshift/os/pull/1001#issuecomment-1245710509 for another link

(I personally still think we should consider some GH actions usage on the openshift/ org in read-only mode, but that's a larger discussion)

Reply:
Yeah. I also had a hacky version of this using a combination of ripgrep, gomplate, and a few other things I'm probably forgetting. Our use-case is a bit different only because we have to render the templates before we can run Shellcheck on them.